packages/components/src/toolbar-group/style.scss

```diff
@@ -21,11 +21,16 @@
 	min-height: $block-toolbar-height;
 	margin: 0;
 	border: $border-width solid $gray-900;
-	border-radius: $radius-block-ui;
```

Comment:
It was used in the legacy toolbar apparently to make toolbar groups rounded, but it seems that it's not needed anymore.